Jenkinsfile.groovy

```diff
@@ -202,54 +202,24 @@ pipeline {
           sh "git fetch --tags origin ${env.BRANCH_NAME}"
 
           if (env.BRANCH_NAME ==~ /release-.*/) {
-            // release
-            sh "npx lerna publish patch \
-            --no-verify-access \
-            --create-release github \
-            --no-commit-hooks \
-            --force-publish \
-            --no-push \
-            --no-git-reset \
-            --yes"
+            sh "node --experimental-specifier-resolution=node build/publishNewVersion.mjs \
+            --bump patch \
+            --tag release \
+            --branch ${env.BRANCH_NAME}"
           } else if (env.BRANCH_NAME == "next") {
-            // next
-            sh "npx lerna publish \
-            --no-verify-access \
-            --conventional-commits \
-            --create-release github \
-            --conventional-prerelease \
-            --preid next \
-            --dist-tag next \
-            --no-commit-hooks \
-            --force-publish \
-            --no-push \
-            --no-git-reset \
-            --yes"
+            sh "node --experimental-specifier-resolution=node build/publishNewVersion.mjs \
+            --bump release \
```

Comment:
do you think this would handle well merging back Next into master? :thinking:

Reply:
I'm guessing that we'll have to do a commit in the PR that merges into master to bump a major